docs/api.md

```diff
@@ -46,7 +46,7 @@
 
 | Name                 | Description                                      |
 |----------------------|--------------------------------------------------|
-| `date_MM_DD_YYYY`    | Defines the content to display in the auro-calendar-cell for the specified date. |
+| `date_MM_DD_YYYY`    | Defines the content to display in the auro-calendar-cell for the specified date. The content text is highlighted green when the `highlight` attribute is used on the slot. |
```

Comment:
We should avoid the word green because it assumes the value of the token used. 
```suggestion
| `date_MM_DD_YYYY`    | Defines the content to display in the auro-calendar-cell for the specified date. The content text is colored using the success state token when the `highlight` attribute is applied to the slot. |
```